gradio/helpers.py

```diff
@@ -448,71 +456,6 @@ async def _start_caching(self):
             else:
                 await self.cache()
 
-    def lazy_cache(self) -> None:
```

Comment:
Why is this being deleted?

Reply:
Previously, we used this method to set up the event listeners in the "lazy caching" case. However, we now use the same event listeners in the "eager" and "lazy" cases -- we just call `load_from_cache` so this function is no longer needed.

Reply:
Ok I missed that previous PR